requirements.txt

```diff
@@ -0,0 +1,3 @@
+invoke
+pyyaml
+ansible-lint
```

Comment:
Let's pin ansible-lint. Having linters unpinned can easily result in getting failed CI runs randomly when new versions are released. I'll open a task to set up dependabot so we would not forget to upgrade.